entity_plus.module

```diff
@@ -227,7 +227,7 @@ function entity_plus_type_supports($entity_plus_type, $op) {
     return in_array('EntityPlusControllerInterface', class_implements($info['controller class']));
   }
   if ($op == 'form') {
-    return (bool) entity_plus_ui_controller($entity_plus_type);
+    return (bool) entity_ui_controller($entity_plus_type);
```

Comment:
@hosef @argiepiano This function (`entity_ui_controller`) is provided by the Entity UI module, which is not a hard requirement for Entity Plus. Do we need to check if the module is enabled, or split this out from Entity UI?

Reply:
@laryn yes, I believe the best approach is to check if Entity UI is enabled. I'll modify

Reply:
I have modified the code to check for module entity_ui

Reply:
Thanks, hopefully either I or @hosef can complete the review of this PR soon and make a new release with your improvements in. Thanks for your work!